docs/guides/contracts.md

```diff
@@ -78,11 +78,10 @@ Note:
         - Key-value map with name of the include as key and source code of the include as value.
     - `validateByteCode` (default: false)
         - Compare source code with on-chain version.
-    - `opt` (default: {})
-        - Object with other [transaction options](../transaction-options.md) which will be provided to **every transaction** that is initiated using the contract instance. You should be aware that:
-            - For most of these additional options it doesn't make sense to define them at contract instance level.
-            - You wouldn't want to provide an `amount` to each transaction or use the same `nonce` which would result in invalid transactions.
-            - For options like `ttl` or `gasPrice` it does absolutely make sense to set this on contract instance level.
+    - other [transaction options](../transaction-options.md) which will be provided to **every transaction** that is initiated using the contract instance. You should be aware that:
```

Comment:
```suggestion
    - an object including other [transaction options](../transaction-options.md) which will be provided to **every transaction** that is initiated using the contract instance. You should be aware that:
```

Reply:
there is no an object at the end of options, transactions options should be fields in `options` object

Reply:
I might lack of javascript knowledge here. so having an `options` object as last param of a function call is usual and typically known to devs? I exactly meant what you wrote as answer. I just wanted to be explicit with the `options` object.

I edited the suggestion. is that better? or can we really leave like is?

Reply:
Looks good to me like this.

Reply:
ahh you mentioned this in the call, right? I am fine with keeping it like is. no big deal :-)

Reply:
> `options` object

is explained above, don't think necessary to repeat it 🙂